tests/RuleSetTest.php

```diff
@@ -772,7 +776,7 @@ private static function assertSameRules(array $expected, array $actual, $message
      *
      * Sometimes keys are all string, sometimes they are integers - we need to account for that.
      */
-    private function sort(array &$data)
+    private function sort(array &$data, array $parentPathParts = [])
```

Comment:
is this change still needed?

Reply:
I forgot about it while fixing merge conflicts... and basically rewrote a similar thing to make tests pass, so yes :)